.github/workflows/backport.yml

```diff
@@ -1,44 +0,0 @@
----
```

Comment:
Why do we want to remove this workflow?

Reply:
I'm not sure if it's used anymore - happy to add it back if it is. I think now the approach we are taking is that the original PR author is responsible for raising a separate PR to previous branches.

Reply:
cc @tmdzk lmk if this is still used!